seed.ts

```diff
@@ -159,6 +162,7 @@ export async function seed() {
 			id: id,
 			username: 'developer',
 			email: 'test@test.com',
+			isAdmin: 1,
```

Comment:
I think the developer account should not be admin. Development should happen from the perspective of a regular user. Locally, we can seed an additional admin account, though.

Reply:
Done :+1: